### PR TITLE
My Tracker app

### PR DESCRIPTION
**Pull request by skabeo**
In this pull request I:

- Creted a splash screen where users can log in or sign up
- On sign up pages users are able to register with their full name and email and password.
- When a user is logged in he is redirected to the category page
  - Where all categories for a particular user shows
  - Total price of all transactions is also displayed
  - The name of the category is displayed on the home screen
  - Users have the ability to add a new transaction
- On the new category form page, users can create a category by entering the category name and providing an icon link
- When a user clicks on a category he is redirected to the category details page where:
  - The name of the category and icon shows
  - The name of the transaction is displayed
  - The total amount of a transaction is also displayed
- A hamburger menu on the categories pages where users can edit their profile and also sign out.

- Here  is  a video presentation on my tracker mobile app. [video](https://drive.google.com/file/d/1xOL2YaSy2KcY_xrjnq8tLdikt8YyqU03/view?usp=sharing)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,183 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a id="support"></a>
+
+If you like this project, please give it a star on GitHub.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a id="acknowledgements"></a>
```
Comment (VICTOR-CHUKWU) [inline README.md:160]: - [ ] The acknowledgement section should contain an appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,183 @@
...
+To run integration tests,run the following command:
+```
+rspec spec/integrations
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Live demo <a id='live'></a>
+
+To view the live demo visit click [here](https://spec-mytracker.onrender.com/)
```
Comment (VICTOR-CHUKWU) [inline README.md:119]: - [ ] for a weird reason I can not signup in  your live demo app kindly check the logs and fix that
Screenshot from 2023-08-09 18-17-21

File: app/views/categories/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+              <p>Add a category</p>
+            <% end %>
+          </div>
+        </a>
+      <% end %>  
+    <% end %>   
+  </div>
+  <a href="<%= new_category_path %>">
+    <%= image_tag 'add.png', class: 'add-item' %>
+  </a>
```
Comment (VICTOR-CHUKWU) [inline app/views/categories/index.html.erb]: - [ ] I must be honest I love your design and your work, but you have to keep to design guidelines, your add button tho beautiful does not look like what we have in the design guide, kindly fix that
your design
image
design guide
Screenshot from 2023-04-27 19-57-40

File: app/views/details/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+          <p class='small-gap'>$ <%= detail.amount %></p>
+        </div>
+      <% end %>
+    <% end %>
+  </div>
+</div>
+
+<a href='/categories/<%= @category.id %>/details/new'>
+  <%= image_tag 'add.png', class: 'add-item' %>
+</a>
```
Comment (VICTOR-CHUKWU) [inline app/views/details/index.html.erb]: - [ ] I must be honest I love your design and your work, but you have to keep to design guidelines, your add button tho beautiful does not look like what we have in the design guide, kindly fix that
your design
image
design guide
Screenshot from 2023-04-27 19-57-40

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @skabeo , 
## STATUS: CHANGES REQUIRED :recycle: 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @skabeo, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title and summary. ✔️
- Ruby on Rails capstone project done successfully ✔️
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**